manual/cli.rst

```diff
@@ -1,4 +1,100 @@
+| -s N       | --seed=N             | use SEED to initialize the pseudo-random generation                                                                           |
++------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| -t STR     | --step=STR           | use STEP to specify how far to proceed in the world generation process. Valid values are: plates precipitations full          |
++------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| -x N       | --width=N            | WIDTH of the world to be generated                                                                                            |
++------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| -y N       | --height=N           | HEIGHT of the world to be generated                                                                                           |
++------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| -q N       | --number-of-plates=N | number of plates                                                                                                              |
++------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------+
+|            | --recursion-limit=N  | you need that just if you encounter an error while generating very large maps                                                 |
++------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| -v         | --verbose            | Enable verbose messages                                                                                                       |
++------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------+
+|            | --version            | Display version information                                                                                                   |
++------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| --bw       | --black-and-white    | Draw maps in black and white                                                                                                  |
```

Comment:
I checked, and the parameter is indeed only used for temp- and precip-map. So I guess it would be appropriate to move it into the "Options valid only for generate"-group in `main.py` instead.